frontend/nuxt.config.ts

```diff
@@ -280,8 +277,7 @@ const config: NuxtConfig = {
     },
   },
   storybook: {
-    port: 6006, // standard port for Storybook
-    stories: ["~/**/*.stories.@(mdx|js)"],
+    port: 6006,
```

Comment:
By default, nuxt storybook scans the `src/components` folder for stories, so we don't need to add this setting here.